lib/ravelin/errors/api_error.rb

```diff
@@ -0,0 +1,19 @@
+module Ravelin
+  class ApiError < StandardError
+    attr_reader :response, :status, :error, :error_message, :validation_errors
+
+    def initialize(response)
+      @response           = response
+      @status             = response.status
+      @error              = response.body.fetch('Error', nil)
+      @error_message      = response.body.fetch('message', nil)
+      @validation_errors  = response.body.fetch('validationErrors', [])
+    end
+
+    def to_s
```

Comment:
Cool, i was expecting `#to_s` to be `#error_message` that's why i was a bit confused here. But i think your explanation / usecase makes sense.